syntax = "proto3";

option java_multiple_files = true;
option java_package = "../downloadarff/";
option java_outer_classname = "DownloadProto";
option objc_class_prefix = "LOG";
package downloadarff;

service Download {
  rpc doDownload (DownloadRequest) returns (DownloadReply) {} // For Download
}

message DownloadRequest {
  string fid = 1;
}

message DownloadReply {
  bytes data = 1;
  int32 status_code = 2; 
}